openmeetings-service/src/main/java/org/apache/openmeetings/service/calendar/caldav/IcalUtils.java

```diff
@@ -226,13 +229,17 @@ private Appointment addVEventPropertiestoAppointment(Appointment a, CalendarComp
 			//If the value of the organizer is an email
 			if ("mailto".equals(uri.getScheme())) {
 				String email = uri.getSchemeSpecificPart();
-				//Contact or exist and owner
-				User org = userDao.getByEmail(email);
-				if (org == null) {
-					org = userDao.getContact(email, a.getOwner());
-					attList.add(createMeetingMember(a, org));
-				} else if (!org.getId().equals(a.getOwner().getId())) {
-					attList.add(createMeetingMember(a, org));
+
+				organizerEmail = email;
+				if(!email.equals(a.getOwner().getAddress().getEmail())) {
```

Comment:
could you modify `if(` to be `if (`  (add space character before `(`)
here and later in the code

Reply:
This has been done for all the files from my side